APMrover2/commands_logic.cpp

```diff
@@ -304,8 +304,8 @@ bool Rover::verify_nav_wp(const AP_Mission::Mission_Command& cmd)
         }
 
         // check if distance to the WP has changed and output new message if it has
-        float dist_to_wp = get_distance(current_loc, next_WP);
-        if ((uint32_t)distance_past_wp != (uint32_t)dist_to_wp) {
+        const float dist_to_wp = get_distance(current_loc, next_WP);
```

Comment:
this isn't a whitespace change either!  Changing code shouldn't be in a whitespace commit.